.github/workflows/python-client.yml

```diff
@@ -33,24 +33,26 @@ jobs:
     strategy:
       matrix:
         go: [ '1.18' ]
+        python: [ '3.8', '3.9', '3.10' ]
```

Comment:
Do we want to test on 3.7? What about adding support for 3.11, has that been tested?